src/main/java/com/minecolonies/client/render/RenderBipedCitizen.java

```diff
@@ -44,6 +47,7 @@
     public RenderBipedCitizen(RenderManager renderManagerIn)
     {
         super(renderManagerIn, defaultModelMale, 0.5F);
+        this.addLayer(new LayerBipedArmor(this));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this call from a constructor to the overridable "addLayer" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1699)